Assets/Scripts/Flocking/LoneFlock.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(FlockAgent))]
+public class LoneFlock : Flock
+{
+    private GameObject target;
+
+    void Start()
+    {
+        this.squareMaxSpeed = this.maxSpeed * this.maxSpeed;
+        this.squareNeighbourRadius = this.neighbourRadius * this.neighbourRadius;
+        SquareAvoidanceRadius = this.squareNeighbourRadius * this.avoidanceRadiusMultiplier * this.avoidanceRadiusMultiplier;
+
+        FlockAgent agent = GetComponent<FlockAgent>();
+        agent.name = ("Self");
```

Comment:
Could I ask why this is named Self?

Reply:
I think Elizabeth added names to flock agents to distinguish agents within a flock, but since a Lone Flock only has 1 agent (itself) I thought it would make sense to imply call it "self".